lms/djangoapps/teams/static/teams/js/spec/edit_team_spec.js

```diff
@@ -0,0 +1,128 @@
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'common/js/spec_helpers/ajax_helpers',
+    'teams/js/views/edit_team'
+], function ($, _, Backbone, AjaxHelpers, TeamEditView) {
+    'use strict';
+
+    describe('EditTeam', function () {
+        var teamEditView,
+            verifyValidation = function (fieldsData) {
+                _.each(fieldsData, function (fieldData) {
+                    teamEditView.$(fieldData[0]).val(fieldData[1]);
+                });
+                teamEditView.$('.create-team.form-actions .action-primary').click();
+                expect(teamEditView.$('.wrapper-msg').is(':visible')).toBeTruthy();
+                expect(teamEditView.$('.wrapper-msg .title').text().trim()).toBe("Oops!");
+                var errorMessage = "Your team could not be created. Check the highlighted fields below and try again.";
```

Comment:
My comment is a "nit" so you don't have to address it, but one way you can inline this while respecting the 120 char limit is to break the line like so:

```javascript
expect(teamEditView.$('.wrapper-msg .copy').text().trim()).toBe(
    "Your team could not be created. Check the highlighted fields below and try again."
);
```